src/main/java/com/metamx/emitter/service/AlertBuilder.java

```diff
@@ -66,9 +67,9 @@ public AlertBuilder severity(AlertEvent.Severity severity)
     return new ServiceEventBuilder<AlertEvent>()
     {
       @Override
-      public AlertEvent build(String service, String host)
+      public AlertEvent build(ImmutableMap<String, String> serviceDimensions)
```

Comment:
Did another way, made `ServiceEventBuilder` an abstract class instead of interface and added bridging method. Should be a safe conversion, especially since `ServiceEventBuilder` is not implemented anywhere outside  emitter repository.